syntax = "proto3";

package config;

import "google/protobuf/empty.proto";

service ConfigService {
    rpc GetConfig (google.protobuf.Empty) returns (ConfigResponse);
}

message ConfigResponse {
    CoinCoeffSettings coin_coeff_settings = 1;
}

message CoinCoeffSettings {
    float maximum = 1;
    float fallback = 2;
}
